Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -41,6 +41,36 @@ public ApplicationPOMModel()
 
         private string mPageURL = string.Empty;
 
+        bool mIsPageLoadBusinessFlow;
+        public bool IsPageLoadBusinessFlow 
+        {
+            get
+            {
+                return mIsPageLoadBusinessFlow;
+            }
+            set
+            {
+                mIsPageLoadBusinessFlow = value;
+                PageLoadFlow = value ? ePageLoadFlow.PageURL : ePageLoadFlow.BusinessFlow;
+                OnPropertyChanged(nameof(this.IsPageLoadBusinessFlow));
+            }
+        }
+
+        ePageLoadFlow mPageLoadFlow;
+        [IsSerializedForLocalRepository]
+        public ePageLoadFlow PageLoadFlow
```

Comment:
please change the name to "PageLoadFlowType"

Reply:
done